### ISSUE TITLE
PyInstaller builds

### ISSUE THREAD

**Issue by hobu** [hobuinc/doppkit#9]
Let's make PyInstaller Windows builds called `doppkit.exe` as part of the build/release process.

I was able to make a successfully working build by adding a `cli.py` file at the top of my tree that did the following:

```
from doppkit  import app

def main():
    """The main function for our script."""

    app.cli()
    
if __name__=='__main__':
    main()


```

And then generating the pyinstaller build:

```
pyinstaller --onefile -c cli.py --name doppkit
```

**Comment by hobu** [comment]
Logo available from https://github.com/hobuinc/doppkit/blob/main/src/grid-icon.ico

**Comment by j9ac9k** [comment]
Should we try and do executables for the other platforms, or just keep it only to Windows?

**Comment by hobu** [comment]
windows-only is sufficient.

**Comment by j9ac9k** [comment]
Can confirm that pyinstaller command works on my machine as well...

**Comment by j9ac9k** [comment]
Fixed with #21
